chainservice/chainservice.go

```diff
@@ -114,6 +114,10 @@ func chainIDmetrics(act action.SealedEnvelope) {
 		_apiCallWithChainIDMtc.WithLabelValues(chainID).Inc()
 	} else {
 		recipient, _ := act.Destination()
+		//it will be empty for staking action, change string to staking in such case
+		if recipient == "" {
+			recipient = "staking"
+		}
```

Comment:
also it might be web3tx, use below to check
```
act, ok := selp.Action().(action.EthCompatibleAction)
ethTx, err := act.ToEthTx()
```
when all these did not check out, classify as "staking"